latex2sympy.py

```diff
@@ -250,6 +250,13 @@ def convert_mp(mp):
             return sympy.MatMul(lh, sympy.Pow(rh, -1, evaluate=False), evaluate=False)
         else:
             return sympy.Mul(lh, sympy.Pow(rh, -1, evaluate=False), evaluate=False)
+    elif mp.CMD_MOD():
+        lh = convert_mp(mp_left)
+        rh = convert_mp(mp_right)
+        if rh.is_Matrix:
+            raise Exception("Cannot perform modulo operation with a matrix as an operand")
+        else:
+            return sympy.Mod(lh, rh)
```

Comment:
@tirkarp I believe we will want to pass `evaluate=False` into Mod, in order to preserve the expression